MovieTime/MovieTime/Model/MTModelMovieDBMovie.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  MTModelMovieDBMovie.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 09/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+struct MTModelMovieDBMovie: Codable {
+
+    var id: Int?
+    var video: Bool?
+    var vote_average: Float?
+    var title: String?
+    var popularity: Float?
+    var poster_path: String?
+    var original_language: String?
+    var original_title: String?
+    var genre_ids: [Int]?
+    var backdrop_path: String?
+    var release_date: Date?
+
+    var overview: String?
+    var runtime: Int?
```

Comment:
About **vote average**, it is missing a better error handling in the view because that data is probably incomplete or incorrect in the API side and it is application responsibility show it properly.
About the **runtime** property, it is possible to be another problem. The request to fetch this data is made after the screen was build, since all data had already been loaded in the `MTMoviesFeedViewController` screen. And no treatment was performed on this failure either.
There are two things that could be improved in the next versions. First, it could have a specific layout for missing/failed information and a new algorithm that tries to reach the information more than once.

Reply:
that algorithm would be 1 line in RxSwift: `retry(x)` (choose the value of x) hehehe